connectors/sources/google_cloud_storage.py

```diff
@@ -128,6 +113,23 @@ def get_default_configuration(cls):
             },
         }
 
+    async def validate_config(self):
+        """Validates whether user inputs are valid or not for configuration field.
+
+        Raises:
+            Exception: The service account json is empty.
+            Exception: The format of service account json is invalid.
+        """
+        if (
+            self.configuration["service_account_credentials"] == ""
+            or self.configuration["service_account_credentials"] is None
+        ):
+            raise Exception("Google Cloud service account json can't be empty.")
+        try:
+            _ = json.loads(self.configuration["service_account_credentials"])
```

Comment:
```suggestion
            json.loads(self.configuration["service_account_credentials"])
```

If you don't need the parsed result, just don't assing it.